packages/terra-core-docs/src/terra-dev-site/doc/toggle-section-header/About.1.doc.mdx

```diff
@@ -10,7 +10,7 @@ import ToggleSectionHeaderPropsTable from 'terra-toggle-section-header/src/Toggl
 
 # Terra Toggle Section Header
 
-Toggle section header component that transitions content in and out with a click.
+Toggle section header component that transitions content in and out with a click. For accessibility best practices, it is recommended that consumers should always use only one h1 tag per page or view. The one h1 tag should be the page title. A section header should never be a heading level 1. 
```

Comment:
Can we move this accessibility info to its own section? Example: https://engineering.cerner.com/terra-ui/components/cerner-terra-core-docs/alert/about#accessibility

Reply:
We have added this text in Toggle section header implementation page which was suggested by UX team as part of this PR. Could you please provide some clarification on this ?

Reply:
I only linked terra-alert as an example for the layout for the component doc. I was asking to create an "Accessibility" section within the Terra Toggle Section Header page and move this text into that section. The 'Terra Toggle Section Header' section (and corresponding sections in other components) seem like they are generally used for a basic description of the component and its typical use, not for prescribing implementation best practices.

Reply:
Updated